zkevm-circuits/src/state_circuit/constraint_builder.rs

```diff
@@ -109,12 +112,23 @@ impl<F: Field> ConstraintBuilder<F> {
             .collect()
     }
 
-    pub fn lookups(&self) -> Vec<Lookup<F>> {
```

Comment:
Offset 0 is constraint by public input, and either it's `Rw::Start` or row propagate from previous chunk.

Therefore, introduce selector in lookup so offset=0 will be skip, due to we already set selector=0 when offset=0